javascripts/discourse/templates/components/right-sidebar-blocks.hbs

```diff
@@ -2,4 +2,8 @@
   <div class={{block.classNames}}>
     {{component block.name params=block.parsedParams}}
   </div>
+  <PluginOutlet
+    @name="below-right-sidebar-block"
```

Comment:
Hmm... if you have multiple blocks, this will render the same component after each block, no? @markvanlan